assets/section-email-signup-banner.css

```diff
@@ -1,3 +1,35 @@
+.email-signup-banner .newsletter-form {
+  display: inline-block;
+}
+
+@media only screen and (max-width: 749px) {
+  .email-signup-banner .banner__box {
+    text-align: center;
+  }
+
+  .banner--mobile-content-align-right .banner__box {
+    text-align: right;
+  }
+
+  .banner--mobile-content-align-left .banner__box {
+    text-align: left;
+  }
+}
+
+@media only screen and (min-width: 750px) {
+  .email-signup-banner .banner__box {
+    text-align: center;
+  }
+
+  .banner--content-align-right .banner__box {
+    text-align: right;
+  }
+
+  .banner--content-align-left .banner__box {
+    text-align: left;
+  }
+}
```

Comment:
Maybe most of this can be in the `section-image-banner.css` file since all section with that similar layout seem to have those settings options 🤔